scripts/build.mjs

```diff
@@ -0,0 +1,88 @@
+import esbuild from "esbuild";
```

Comment:
It seems the indentation is wonky in this file

Reply:
`npm run lint` wasn't checking the `build.mjs` file. I've updated `package.json` accordingly and fixed the linting!